src/api/files/service.js

```diff
@@ -397,6 +410,10 @@ async function getAndVerify(fileId, retrievalKey) {
     throw Boom.notFound('File not found')
   }
 
+  logger.info(
+    `Comparing retrieval keys - incoming: ${retrievalKey}, stored: ${fileStatus.retrievalKey}`
+  )
```

Comment:
Don't log this, treat it as a credential

Reply:
Other than that it's good to go

Reply:
😱  Oversight on my part- my punishment for not setting up the debugger properly